Injector.java

```diff
@@ -0,0 +1,78 @@
+package mate.academy.di;
+
+import mate.academy.dao.*;
+import mate.academy.factory.ClientDaoFactory;
+import mate.academy.factory.HumanDaoFactory;
+import mate.academy.handler.ClientConsoleHandler;
+import mate.academy.handler.HumanConsoleHandler;
+
+import java.lang.reflect.Field;
+import java.util.Scanner;
+
+public class Injector {
+
+    public static void create() throws IllegalAccessException {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Выберете объект: 1 - Клиент,  2 - Человек, 3 - для вихода");
+        int a = scanner.nextInt();
+        switch (a) {
+            case 1:
+                injectDependencyForClient();
+                break;
+            case 2:
+                injectDependencyForHuman();
+                break;
+            default:
+                return;
+        }
+
+    }
+
+    private static void injectDependencyForClient() throws IllegalAccessException {

```

Comment:
Normally this method should scan everything for annotation, currently it is bounded to 3 classes ClientConsoleHandler, FileClientDao, InMemoryClientDao and when you will add more classes you will need to modify this method

Reply:
> Normally this method should scan everything for annotation, currently it is bounded to 3 classes ClientConsoleHandler, FileClientDao, InMemoryClientDao and when you will add more classes you will need to modify this method

Yes, it is a problem, but I do not know how to solve it

Reply:
Simple workaround is to pass list of classes to method or store it as field, and make this method inject everything to this classes, this should be not hard to implement